[#5006] Ability to manually fill in `addressNL` streetname and city

vaszig opened the pull request:
Closes #5006

- Needs

  - [x] new version of form-builder
  - [ ] frontend translations

**Changes**

- Updated street name and city serializer fields in order to be able to require these fields in case 
the component is required.


**Checklist**

Check off the items that are completed or not relevant.

- Impact on features

  - [x] Checked copying a form
  - [x] Checked import/export of a form
  - [x] Config checks in the configuration overview admin page
  - [x] Problem detection in the admin email digest is handled

- Release management

  - [x] I have labelled the PR as "needs-backport" accordingly

- I have updated the translations assets (you do NOT need to provide translations)

  - [x] Ran `./bin/makemessages_js.sh`
  - [x] Ran `./bin/compilemessages_js.sh`

- Dockerfile/scripts

  - [x] Updated the Dockerfile with the necessary scripts from the `./bin` folder

- Commit hygiene

  - [x] Commit messages refer to the relevant Github issue
  - [x] Commit messages explain the "why" of change, not the how

sergei-maertens commented on src/openforms/formio/components/custom.py:
```diff
@@ -457,18 +481,23 @@ def validate_postcode(self, value: str) -> str:
-            existing_hmac = attrs.get("secretStreetCity", "")
-            postcode = attrs.get("postcode", "")
-            number = attrs.get("houseNumber", "")
+            # when the user fills in manually the city and the street name we do not
+            # need to check the secret city - street name combination
+            if not auto_populated:
+                return attrs
```
ah okay, and because `autoPopulated = False` we then also know in the registration backend that the data can potentially not be trusted, right?

vaszig replied:
Yes, this is how I thought about it.Let me know if you agree.

vaszig committed:
[#5006] Updated AddressValueSerializer to adapt manual filling of streetname and city

Updated street name and city serializer fields in order to be able to
require these fields in case the component is required.

vaszig commented:
Missing the dutch translations for this..
